app/stack/log-drains/new/route.js

```diff
@@ -37,6 +37,7 @@ export default Ember.Route.extend({
 
     createLog(log) {
       log.set('stack', this.modelFor('stack'));
+      log.set('status', 'provisioning'); // Ensures polling for status updates
```

Comment:
This is how I got the polling to work. Or we could added polling on a `pending` status as well in ember-shared.